src/options.html

```diff
@@ -2,7 +2,9 @@
 <meta charset="utf-8">
 <title>Refined GitHub options</title>
 <style>
-	html {
+	body {
+		width: 350px;
+		margin: 14px 17px; /* Chrome default, match for Firefox */
```

Comment:
Maybe open an issue on Firefox about this? I know it's minor, but ideally, you shouldn't have to do custom things for Firefox.